src/Stream/test/BinderRabbitMQ.Test/RabbitBinderTests.cs

```diff
@@ -1843,12 +1892,26 @@ public void TestPolledConsumerWithDlqRePub()
             binding.Unbind();
         }
 
-        [Fact]
-        public void TestAnonymousGroup()
+        // TestCustomBatchingStrategy - not supported in Steeltoe
```

Comment:
Is this comment related to a test that wasn't ported? Without context I don't understand what it's here for